migrator/src/main/scala/com/scylladb/migrator/Validator.scala

```diff
@@ -160,6 +46,7 @@ object Validator {
     else {
       log.error("Found the following comparison failures:")
       log.error(failures.mkString("\n"))
+      System.exit(1)
```

Comment:
this works for standalone mode, what about yarn mode? (I think EMR has it like that)
will above trigger failing the app with unrecoverable error in yarn mode?

Reply:
also above you just do sys.error ... (hence my confusion)

Reply:
Is there a way to test the Yarn mode? It seems that indeed our testing infrastructure uses the [standalone mode](https://hub.docker.com/r/bde2020/spark-master).

Reply:
I added this line to have a way to check in the tests whether the validation [succeeded](https://github.com/julienrf/scylla-migrator/blob/6a20c570444f07aaf6c38c3b55b4c7da3ef9f6f5/tests/src/test/scala/com/scylladb/migrator/alternator/ValidatorTest.scala#L27) or [failed](https://github.com/julienrf/scylla-migrator/blob/6a20c570444f07aaf6c38c3b55b4c7da3ef9f6f5/tests/src/test/scala/com/scylladb/migrator/alternator/ValidatorTest.scala#L41).

Reply:
this checks for process exit code, so we will need to test this, let's merge and file new bugs for that then